src/js/components/Select/__tests__/__snapshots__/Select-test.js.snap

```diff
@@ -5905,6 +5909,8 @@ exports[`Select keyboard navigation timeout 1`] = `
   border-radius: 4px;
   outline: none;
   border: none;
+  opacity: 0.3;
```

Comment:
It seems suspicious that this styling is being added here even though nothing changed in select-test.